### ISSUE TITLE
AdamWeightDecayStrategy学习率问题

### ISSUE THREAD

**Issue by TyrionZK** [PaddlePaddle/PaddleFormers#857]
文本分类中使用paddlehub对bert二次训练时，使用了AdamWeightDecayStrategy，参数如下：
warmup_proportion 0.2
weight_decay 0.01
learning_rate 2e-5，
问题：我发现每当新的一个epoch开始的时候，数据震荡很大，导致后面几个epoch没法对数据继续进行提高。我怀疑AdamWeightDecayStrategy是不是针对每个epoch调整学习率，导致每个epoch开始都会经历学习率增大，然后减小的呢？
哪位大佬能给解释一下吗，谢谢。

**Comment by Steffy-zxf** [comment]
你好！关于AdamWeightDecayStrategy策略可以参考 https://aistudio.baidu.com/aistudio/projectdetail/185121 中策略说明。该策略的学习率变化，是根据总共需要训练的step数变化，不随着一个epoch变化。经线下沟通，可以尝试 weight decay 加大，learning rate 调小。

**Comment by TyrionZK** [comment]
好的，谢谢

**Comment by TyrionZK** [comment]
好的，谢谢
